lang-ocaml/ocaml-fpath/autobuild/build

```diff
@@ -0,0 +1,5 @@
+abinfo "Building ocaml-fpath ..."
+ocaml pkg/pkg.ml build --dev-pkg false
```

Comment:
Absolute path for `pkg/pkg.ml`?

Reply:
Also, possible candidate for a template?

Reply:
Fixed